apps/cowswap-frontend/src/modules/twap/containers/TwapFormWarnings/warnings/UnsupportedWalletWarning.tsx

```diff
@@ -23,6 +23,9 @@ export function UnsupportedWalletWarning({ isSafeViaWc }: { isSafeViaWc: boolean
         <ExternalLink href={UNSUPPORTED_WALLET_LINK}>Click here</ExternalLink>. Future updates may extend wallet
         support!
       </p>
+      <p>
+        <strong>Note:</strong> If you are using a Safe but still see this message, ensure your Safe is deployed!
+      </p>
```

Comment:
Cool, but it is very likely the underlying issue is us, relying on checking that the address has code in it, to decide if it's a smart contract wallet. 

I would suggest making the decision based on the provider being used: If it's safe provider, it's Safe, and we mark the `isSmartContractWallet: true`